src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -2063,9 +2085,50 @@ abstract class DatabaseTest {
             endDate = endDate,
         )
 
+    nextCohortModuleStartDate = endDate.plusDays(1)
```

Comment:
Would it make sense to add 7 days instead of 1 day here, so you can insert two cohort modules in a row using default start/end dates without overlaps?

Reply:
This uses end date, so it actually will never overlap even if we said a custom end date.

Reply:
Oops, right you are! Ignore my comment, then.